src/manual-inline/index.scss

```diff
@@ -0,0 +1,35 @@
+@import '_core';
+@import '_adverts';
+@import '_advert';
+@import '_adverts-books';
+@import '_adverts-brand';
+@import '_adverts-jobs';
+@import '_adverts-subscriptions';
+@import '_adverts-networks';
+@import '_adverts-soulmates';
+@import '_adverts-live';
+@import '_adverts-membership';
+@import '_adverts-masterclasses';
+@import '_adverts-travel';
+@import '_adverts-shop';
+@import '_adverts-money';
+@import '_adverts-gardening';
+
+a.adverts__logo {
```

Comment:
Instead of doing that, you can just add the `blink` class to the `DIV.advert__logo` and then surround the text inside with `<span class="blink__anchor"></span>` (see other templates for ref)